modular_pentest/modules/robots/serv_borg.dm

```diff
@@ -0,0 +1,363 @@
+		"Default - 'Hydro'" = image(icon = 'icons/mob/robots.dmi', icon_state = "hydrobot"),
+		"Marina" = image(icon = 'icons/mob/robots.dmi', icon_state = "marinaSV"),
+		"Sleek" = image(icon = 'icons/mob/robots.dmi', icon_state = "sleekservice"),
+		"Kodiak - 'Teddy'" = image(icon = 'icons/mob/robots.dmi', icon_state = "kodiak-service"),
+		"Noble" = image(icon = 'icons/mob/robots.dmi', icon_state = "Noble-SRV"),
+		"R34 - SRV9a 'Llyod'" = image(icon = 'icons/mob/robots.dmi', icon_state = "lloyd"),
+		"Meka Service - F" = image(icon = 'modular_pentest/icons/mob/numekas1.dmi', icon_state = "fmekaserv"),
+		"Meka Service - M" = image(icon = 'modular_pentest/icons/mob/numekas1.dmi', icon_state = "mmekaserv"),
+		)
+		service_icons = sortList(service_icons)
+	var/service_robot_icon = show_radial_menu(R, R , service_icons, custom_check = CALLBACK(src, PROC_REF(check_menu), R), radius = 42, require_near = TRUE)
+	switch(service_robot_icon)
+		if("Default - 'Waitress'")
+			cyborg_base_icon = "service_f"
+		if("Default - 'Butler'")
+			cyborg_base_icon = "service_m"
+		if("Default - 'Bro'")
+			cyborg_base_icon = "brobot"
+		if("Default - 'Maximillion'")
+			cyborg_base_icon = "tophat"
+			special_light_key = null
+			hat_offset = INFINITY //He is already wearing a hat
+		if("Default - 'Hydro'")
+			cyborg_base_icon = "hydrobot"
+			cyborg_icon_override = 'icons/mob/robots.dmi'
+			special_light_key = "hydrobot"
+		if("Default - 'Kent'")
+			cyborg_base_icon = "kent"
+			special_light_key = "medical"
+			hat_offset = 3
+		if("Marina")
+			cyborg_base_icon = "marinaSV"
+			cyborg_icon_override = 'icons/mob/robots.dmi'
+			special_light_key = "marinaSV"
+		if("Sleek")
+			cyborg_base_icon = "sleekservice"
+			cyborg_icon_override = 'icons/mob/robots.dmi'
+			special_light_key = "sleekservice"
+		if("Kodiak - 'Teddy'")
+			cyborg_base_icon = "kodiak-service"
+			cyborg_icon_override = 'icons/mob/robots.dmi'
+			special_light_key = "kodiak-service"
+		if("Noble")
+			cyborg_base_icon = "Noble-SRV"
+			cyborg_icon_override = 'icons/mob/robots.dmi'
+			special_light_key = "Noble-SRV"
+		if("R34 - SRV9a 'Llyod'")
+			cyborg_base_icon = "lloyd"
+			cyborg_icon_override = 'icons/mob/robots.dmi'
+			special_light_key = "lloyd"
+		if("Meka Service - F")
+			cyborg_base_icon = "fmekaserv"
+			cyborg_icon_override = 'modular_pentest/icons/mob/numekas1.dmi'
+			special_light_key = "fmekaserv"
+		if("Meka Service - M")
+			cyborg_base_icon = "mmekaserv"
+			cyborg_icon_override = 'modular_pentest/icons/mob/numekas1.dmi'
+			special_light_key = "mmekaserv"
+		else
+			return FALSE
+	return ..()
+
+/*
+		SERVICE BORG UPGRADES
+
+		See robot_upgrades.dm
+*/
+
+
+//Condiment Synthesizer
+/obj/item/borg/upgrade/condiment_synthesizer
+	name = "Service Cyborg Condiment Synthesiser"
+	desc = "An upgrade to the service model cyborg, allowing it to produce solid condiments."
+	icon_state = "cyborg_upgrade3"
```

Comment:
FYI this is just a temp place holder until the _robot_upgrades.dm is finished... Will be swapped with the other icon state at that time.